_posts/2024-01-03-opensearch-performance-improvements.md

```diff
@@ -0,0 +1,1141 @@
+
+### Storage investments
+
+Storage is another major factor that affects the overall efficiency of log analytics and search workloads. In OpenSearch 2.9 and later, customers can use Zstandard compression, resulting in a 30% reduction in on-disk data size while maintaining a near-identical CPU utilization pattern compared to the default compression. Some of the ongoing work, such as addition of a `match_only_text` field (see [ #11039](https://github.com/opensearch-project/OpenSearch/pull/11039)) has shown promising improvements of about 25% reduction for data on disk, primarily with text data field optimization, and should be available for users in the upcoming OpenSearch 2.12 release. 
+
+## Measured performance improvements
+
+To compare the performance between Elasticsearch 7.10.2 and OpenSearch 2.11, we ran query operations for widely used scenarios in log analytics, time series, and search. We ran the queries across clusters running each version and documented the resulting performance. The following are the key findings from this exercise.
+
+### Log analytics
+
+For log analytics use cases, we used the `http_logs` workload from the [OpenSearch Benchmark](https://opensearch.org/docs/latest/benchmark/index/)---a macro-benchmark utility within the OpenSearch Project---to replicate some of the common query operations. Here are the key highlights:
+
+* `match_all` queries with sorting showed a more than 20x performance boost across the board because of multiple improvements made in the area (see [#6321](https://github.com/opensearch-project/OpenSearch/pull/6321) and [#7244](https://github.com/opensearch-project/OpenSearch/pull/7244)) and other Lucene enhancements.
+
+* Queries for ascending and descending sort-after-timestamp saw a significant performance improvement of up to 70x overall. The optimizations introduced (such as [#6424](https://github.com/opensearch-project/OpenSearch/pull/6424) and [#8167](https://github.com/opensearch-project/OpenSearch/issues/8167)) extend across various numeric types, including but not limited to `int`, `short`, `float`, `double`, `date`, and others.
+
+* Other popular queries such as `search_after` saw an about 60x reduction in latency, attributed to the improvements made in the area involving optimally skipping segments during search (see [#7453](https://github.com/opensearch-project/OpenSearch/pull/7453)). The `search_after` queries can be used as the recommended alternative to scroll queries for a better search experience.
```

Comment:
Can we add a line item after:

* Implementation support for match_only_text field to optimize on storage and indexing/search latency for text queries is in progress ([#11039](https://github.com/opensearch-project/OpenSearch/pull/11039)).